source/index.ts

```diff
@@ -73,9 +73,22 @@ class Chromium {
     } else {
       const input = join(__dirname, '..', 'bin');
       const promises = [
-        inflate(folder, `${input}/chromium.br`),
-        inflate(folder, `${input}/swiftshader.tar.br`),
-        inflate(folder, `${input}/aws.tar.br`),
+        inflate({
+          folder,
+          file: `${input}/chromium.br`
+        }),
+        // Swiftshader needs to be extracted at the same level as chromium and not inside a swiftshader director
+        // https://github.com/alixaxel/chrome-aws-lambda/pull/264#issuecomment-1136311984
+        inflate({
+          folder,
+          file: `${input}/swiftshader.tar.br`,
+          targetFolder: folder,
+          checkFileToExists: `${folder}/libGLESv2.so`
```

Comment:
I think it's only to avoid unpacking the `Swiftshader.tar.br` file every time we use the `prepare` method (in our lambda-runners it happens in the `setup()` function in the Playwright runtime and also every time a user calls `browser.launch()`).
Given we extract each Swiftshader libs in the root directory (for our runners it's `/tmp/2022-02/playwright`, `/tmp/2021-10/playwright `etc...) we need to check if one of those libs exists to avoid extracting Swiftshader again (actually we check for this file but it doesn't matter much, we only want to make sure Swiftshader is extracted).

Reply:
Gotcha

Reply:
Btw, we don't unpack it each time. If you look above, we check if chromium exists and if so, we return it

Reply:
I agree but that's why we're checking that `libGLESv2.so` exists, to avoid unpacking it each time (like we do with chromium)